package.json

```diff
@@ -17,34 +17,24 @@
     "prepublishOnly": "npm run build",
     "prebuild": "rimraf dist",
     "build": "babel src --out-dir dist --copy-files --ignore **/*.test.js",
-    "lint": "xo",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:js": "eslint . --ext js,ts --cache",
+    "lint:js:fix": "eslint . --ext js,ts --fix",
```

Comment:
I'd probably just have the `lint:js` script, but just name it lint and dump all the others. Using npm-run-all is usually used if you run multiple linters in a project. We're only running one, so we don't really need all that extra stuff.